src/openfl/display/DisplayObject.hx

```diff
@@ -978,6 +1049,14 @@ class DisplayObject extends EventDispatcher implements IBitmapDrawable #if openf
 	}
 	
 	
+	private function __setWorldTransformValid ():Void {
+		
+		__worldTransformInvalid = false;
+		__transformDirty = false;
+		
+	}
```

Comment:
Having an `Invalid` and a `Valid` function that are setting different variables feels not right to me. Because it's just used in two places, maybe we can simply reset both variables there?

While thinking more about it, actually the question is what "transform dirty" actually means. I think ideally we would have flags for everything that might become dirty and really only update what has changed (e.g. world transform, render transfrom, alpha, color transform, blend mode, ...). Anyhow, this is a different discussion...

Maybe it could be nice to reset both variables in `__updateTransform`, but than we would need to find a good way to not do it on `__updateRenderData`... Well, overall I think I don't have a great idea in mind except that I think this function is not ideal `;)`